### ISSUE TITLE
Linkability is broken in the lite branch

### ISSUE THREAD

**Issue by LOuroboros** [froggestspirit/Emerald20XX#14]
Title.
Tested using VBA 1.7.2.
Linkability in the Master branch works just fine.
I'm guessing this is a result of removing wireless connection related code and other things from the lite branch.
I'm opening this Issue ticket because I'm not fully sure if linking is supposed to be broken in that branch or not.

https://user-images.githubusercontent.com/4485172/144486602-de14011c-8b28-48a3-b5f4-97d7dd37ae67.mp4

Here's the .sa1 and .sa2 files to load up in VBA 1.7.2 *(File -> Import -> Battery File)*:
[em20xx_lite.zip](https://github.com/froggestspirit/Emerald20XX/files/7644477/em20xx_lite.zip)

**Comment by froggestspirit** [comment]
Please see if the glitch exists in MGBA, VBA id known to not work as
reliably

**Comment by froggestspirit** [comment]
Though it might be an issue with the lite branch if master is working...

**Comment by LOuroboros** [comment]
> Please see if the glitch exists in MGBA, VBA id known to not work as

Yeah, I'm aware. It's just that I never really tested linking on mGBA before. Oh well, there's a first time for everything.

I'll edit in a couple of minutes with the results.

**Comment by froggestspirit** [comment]
Sorry about that, i saw VBA and kind of jumped the gun a little

**Comment by LOuroboros** [comment]
I'll be completely honest, I'm not sure if mGBA's linking capabilities are even working at all, like, officially? 😂 ...

It gets stuck on "Please wait." so it's not working regardless, but I can't tell if mGBA is the one to blame in this situation, or not.

https://user-images.githubusercontent.com/4485172/144493160-45b2740e-9a7a-4950-b941-79fedc28b236.mp4

**Comment by froggestspirit** [comment]
Its completely possible i broke it lol. Hard to tell sometimes what can
stay or go. I dont think the lite branch provides significant advantages
over master, so it might be better to use that for the time being

**Comment by LOuroboros** [comment]
Alright, I got it.

It doesn't work.

If you try to trade between 2 different savefiles, you get the same error that VBA Link gives.

If you try to trade between 1 same savefile, you can get a bit further, but an error kills the connection just before starting a trade. 

https://user-images.githubusercontent.com/4485172/144495288-47b03dfc-62c8-4e6d-b79f-494b09dbb557.mp4

https://user-images.githubusercontent.com/4485172/144494549-c4c5751a-eb6d-4e99-b1b4-1afd8e65d168.mp4

**EDIT**:

> I dont think the lite branch provides significant advantages
> over master, so it might be better to use that for the time being

Actually, I'm manually porting stuff over to my project. I'm not using it as a base 😅 

I'm guessing the removal of the wireless connection or some of its components is what could have caused this problem?

I'll do some tests to try to find out.
